src/govuk/components/notification-banner/notification-banner.yaml

```diff
@@ -0,0 +1,134 @@
+params:
+- name: title
+  type: string
+  required: false
+  description: Custom title text. Defaults to 'Important' ('Success' for success banner and 'Error' for error banner). If `titleHtml` is supplied, this is ignored.
+- name: titleHtml
+  type: string
+  required: false
+  description: Custom title HTML. Defaults to 'Important' ('Success' for success banner and 'Error' for error banner).
+- name: titleHeadingLevel
+  type: string
+  required: false
+  description: Heading level, from 1 to 6. Default is `2`.
+- name: content
+  type: string
+  required: true
+  description: Content for the notification banner. If `contentHtml` is supplied, this is ignored.
+- name: contentHtml
+  type: string
+  required: true
+  description: HTML content for the notification banner.
+- name: type
+  type: string
+  required: false
+  description: Sets classes and attributes to create a type of notification banner. Accepts `success` or `alert`.
+- name: role
+  type: string
+  required: false
+  description: Custom role for the notification banner. Defaults to `region` (`alert` for success and error banners).
+- name: tabindex
+  type: string
+  required: false
+  description: Custom tabindex for the notification banner. Not set by default (defaults to `-1` for success and error banners).
+- name: initialFocus
+  type: string
+  required: false
+  description: Sets `data-module="govuk-initial-focus"` for the notification banner which focuses the banner on page load. Not set by default (on by default for success and error banners).
```

Comment:
Is it useful to mention the data attribute here? Should this just describe the outcome ("Focus the banner when the page loads")?

Reply:
Yes it's on my list to make these explanations more helpful 👍

Reply:
Again I've tried to clarify the documentation here but will review it with Eoin next.